install/install.ps1

```diff
@@ -0,0 +1,65 @@
+# Enable a default -Verbose flag for debug output.
+[CmdletBinding()]
+Param()
+
+# Halt script execution at the first failed command.
+$script:ErrorActionPreference='Stop'
+
+# We don't currently have good cross-platform options for determining the latest release version, so we require
+# knowledge of the specific target version, which the user should set as an environment variable.
+$version = $Env:AZUREAUTH_VERSION
+if ($null -eq $version) {
+    # Write-Error is terminal with ErrorActionPreference='Stop', so we continue to hit the exit
+    # and set an exit code.
+    Write-Error 'No $AZUREAUTH_VERSION specified, unable to download a release' -ErrorAction:Continue
+    exit 1
+}
+

```

Comment:
I am trying to confirm this catches my not having the env var set but it doesn't seem to work in PS5??

```
❯ iex "& { $(irm https://raw.githubusercontent.com/AzureAD/microsoft-authentication-cli/install-scripts/install/install.ps1) } -Verbose"
iex : At line:18 char:41
+ $repo = ($null -eq $Env:AZUREAUTH_REPO) ? 'AzureAD/microsoft-authenti ...
+                                         ~
Unexpected token '?' in expression or statement.
At line:1 char:1
+ iex "& { $(irm https://raw.githubusercontent.com/AzureAD/microsoft-au ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ParserError: (:) [Invoke-Expression], ParseException
    + FullyQualifiedErrorId : UnexpectedToken,Microsoft.PowerShell.Commands.InvokeExpressionCommand
```

Reply:
`?` was added in PS 7, but not in 5.
https://stackoverflow.com/questions/31341998/ternary-operator-in-powershell